style.css

```diff
@@ -0,0 +1,236 @@
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  overflow-x: hidden;
+}
+
+/* Mobile Styles */
+@media screen and (min-width: 375px) {
+  .header {
```

Comment:
- [ ] great job so far. please remember your design should look alike with the figma version. kindly add necesary margins and paddings so it can follow the design like the previous reviewer suggested. 

your design             |  original design
:-------------------------:|:-------------------------:
![Screenshot (1018)_LI](https://user-images.githubusercontent.com/65298048/235072786-41f9f9fe-4e10-4a73-9d01-165f0ac78f42.jpg)  |  ![Screenshot (1016)](https://user-images.githubusercontent.com/65298048/235072910-14a905e0-2b6b-4ee8-b520-f876f0c6d0e1.png)